src/app/client/src/app/modules/workspace/components/content-editors/generic-editor/generic-editor.component.ts

```diff
@@ -101,6 +102,26 @@ export class GenericEditorComponent implements OnInit, OnDestroy {
       env: 'generic-editor',
       framework: this.routeParams.framework
     };
+
+    // Fetching ownership type
+    const formServiceInputParams = {
```

Comment:
This is DRY code please have single place to get the form data and use it and will editor wait until the form api is send response back or it loads and what happens to the config if the editor loads without form api response